src/pinyin.cc

```diff
@@ -44,7 +44,7 @@ std::set<std::string> PinYin::to_plain(const std::string &input) {
 
 // clang-format off
 std::map<int, std::vector<std::string> > PinYin::build_pinyin_map() {
-  std::map<int, std::vector<std::string> > pinyin;
```

Comment:
Renamed this local `pinyin` variable to `map` since it's collided with `PinYin::pinyin`